pkg/plugins/common/kustomize/v2/plugin.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v2
+
+import (
+	"sigs.k8s.io/kubebuilder/v3/pkg/config"
+	cfgv3 "sigs.k8s.io/kubebuilder/v3/pkg/config/v3"
+	"sigs.k8s.io/kubebuilder/v3/pkg/model/stage"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugins"
+)
+
+// KustomizeVersion is the kubernetes-sigs/kustomize version to be used in the project
+const KustomizeVersion = "v4.5.3"
```

Comment:
Hi @rashmigottipati thank you for looking on that: 
I added a comment on the plugins/go/v2 code that clarifies why we cannot update there:

```
// @Deprecate: KustomizeVersion came from the Kustomize plugin since go/v3
	// Go/v2 plugin exist only to ensure the backwards compatibility with the old scaffold
	// produced with kubebuilder >= 2.0.0 < 3.0.0. It does not really take advantage of the plugin system
	// (see that the PROJECT file does not have the plugins layout and uses the old/legacy
	// version/schema.) This plugin will be deprecated with go/v4-alpha and can be removed
	// when go/v4 stable be publish.
```

Also, see: https://github.com/kubernetes-sigs/kubebuilder/pull/2583/files#r839541438

The kustomize version should be in the kustomize plugin from go/v3 we sorted it out.

Reply:
Thank you for the explanation. That makes sense.